Tests/FBLPromisesTests/FBLPromise+ReduceTests.m

```diff
@@ -54,7 +54,7 @@ - (void)testPromiseReduce {
 - (void)testPromiseReduceReject {
   // Arrange.
   NSArray<NSNumber *> *numbers = @[ @1, @2, @3 ];
-  NSInteger __block count = 0;
+  NSUInteger __block count = 0;
```

Comment:
Intentional? Signed count seems fine everywhere.

Reply:
For the most part, we've been using NSUInteger for ObjC "counts" in test methods. Made the change here so we are being consistent across all tests.